JavaScript/JSON/Types/Instances.hs

```diff
@@ -1064,15 +1063,6 @@ realFloatToJSON d
     | otherwise               = doubleValue (realToFrac d)
 {-# INLINE realFloatToJSON #-}
 
-scientificToNumber :: Scientific -> Number
```

Comment:
I don't remember this too well. I see that it was used in an instance that's commented out, and it's not exported - so I'd guess that I removed it because of an unused function warning.